cypress/e2e/safe-apps/tx-builder.spec.cy.js

```diff
@@ -146,8 +146,8 @@ describe('Tx-builder Safe App tests', { defaultCommandTimeout: 20000 }, () => {
       getBody().findByText(safeapps.createBatchStr).click()
       getBody().findByText(safeapps.sendBatchStr).click()
     })
-    cy.get('p').contains('1').should('be.visible')
-    cy.get('p').contains('2').should('be.visible')
+    cy.get('p').contains('1').should('exist')
+    cy.get('p').contains('2').should('exist')
```

Comment:
This was failing as in https://github.com/safe-global/safe-wallet-web/pull/2768#discussion_r1386237622.